src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1312,7 +1312,7 @@ def test_format_cli_exception(self, capsys):
             outerr = capsys.readouterr()
             assert outerr.err
             error_msg = json.loads(outerr.err)
-            assert error_msg["code"] == "ConnectionNotFoundError"
+            assert error_msg["code"] == "SDKError"
```

Comment:
I think there is a bug in the code that caused the **error_msg["code"]** to be correct, but the logic is quite complex. I won't modify it in this PR, so the colleague who wrote this part of the code needs to determine whether it is a bug or a special business logic.

![image](https://github.com/microsoft/promptflow/assets/23182548/21fcde17-fd90-456a-b511-38dbc4021ef2)
![image](https://github.com/microsoft/promptflow/assets/23182548/5af0d50e-b2c8-42b7-8ffa-9b5f32c714de)